src/modules/layout/components/Header.js

```diff
@@ -160,14 +160,15 @@ class Header extends React.PureComponent<PropsType> {
     return <BoxShadow theme={theme}>
       <Horizontal>
         <HorizontalLeft>
-          {this.canGoBackInStack() ? <HeaderBackButton onPress={this.goBackInStack} /> : <Logo source={logo} />}
+          {this.canGoBackInStack() ? <HeaderBackButton onPress={this.goBackInStack} />
+          : <Logo source={logo} accessibilityLabel={'Integreat'} />}
```

Comment:
Maybe `Home` or `Dashboard` would be better here.

Reply:
@maxammann @Schedulaar what do you think?

Reply:
Does this always go to the dashboard?

Reply:
I tink so, yes

Reply:
No it's only display when we're already on the Dashboard. So either we use an empty alt text or 'Integreat Logo'. (But probably rather empty)

Reply:
Rather empty those thumb ups are for :)